Wireframe/index.html

```diff
@@ -3,30 +3,67 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
+    <title>SHIRAZ</title>
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
+      <h1>SHIRAZ</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        This is the picture of Shiraz my beautiful city.
       </p>
+
     </header>
+
     <main>
+      
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src="shiraz3.jpg" alt="" />
+        <h2></h2>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
+           What is the purpose of a README file?The readme file is used to explain what is uploaded and how we can install or use it.
+         </p>
+         <P>What is the purpose of a wireframe?A wireframe is commonly used to layout content and functionality on a page which takes into account user needs and user journeys.  
```

Comment:
Some of these paragraph tags are lower-case (`<p>`) and some upper-case (`<P>`) - is there a difference between them? Why did you choose to do these differently? We tend to default to consistency when writing code unless there's a reason to be inconsistent.